plugin/trino-prometheus/src/test/java/io/trino/plugin/prometheus/PrometheusServer.java

```diff
@@ -45,7 +45,8 @@ public PrometheusServer(String version, boolean enableBasicAuth)
         this.dockerContainer = new GenericContainer<>("prom/prometheus:" + version)
                 .withExposedPorts(PROMETHEUS_PORT)
                 .waitingFor(Wait.forHttp(METRICS_ENDPOINT).forResponsePredicate(response -> response.contains("\"up\"")))
-                .withStartupTimeout(Duration.ofSeconds(120));
+                .waitingFor(Wait.forHttp("/api/v1/query?query=up[1d]").forResponsePredicate(response -> response.contains("\"values\"")))
```

Comment:
Please replace L47 instead of adding a new line. Also, please fix inside of `enableBasicAuth` too in a separated commit.

Reply:
Done